src/CMakeLists.txt

```diff
@@ -60,6 +60,13 @@ else()
     message(WARNING "Hyrise will be built with most compiler warnings deactivated. This is fine if you want to test Hyrise but will become an issue when you want to contribute code.")
 endif()
 
+if (CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+     if (CMAKE_CXX_COMPILER_VERSION VERSION_GREATER_EQUAL 13 AND CMAKE_CXX_COMPILER_VERSION VERSION_LESS_EQUAL 13.99)
+    message(WARNING "Disabling 'reserved-identifier' warning for clang 13 (version is ${CMAKE_CXX_COMPILER_VERSION}).")
```

Comment:
This line should be more indented than the previous line and on the same indention level as the following line.

Reply:
Wow, that looked terrible. No idea how that happened.

Fixed.